Nucleation_model.py

```diff
@@ -0,0 +1,89 @@
+#%%
+import datetime as dt
+import shutil
+import math as m
+import numpy as np
+import pandas as pd
+import random as ran
+import matplotlib.pyplot as plt
+from matplotlib.backends.backend_pdf import PdfPages
+from scipy.integrate import solve_ivp
+#%%
+"""=========================== Constants ==========================="""
+
+k_B = 1.380649E-23 #J K-1 // Boltzmann's constant
+ep_0 = 8.854E-12  #F m-1 // electric permittivity of a vacuum
+pi = 3.141529
+N_a = 6.022E23 #mol-1 // Avogadro's number
+R = 8.3145 #J mol-1 K-1 // Ideal constant
+
+"""=========================== User system variables ==========================="""
```

Comment:
I call out a few specific varaibles, below, but in general, I would like comments to give slightly more detailed information about specifically what each input variable represents.